### PR TITLE
Smashing Magazine 

### PR DESCRIPTION
**Pull request by BrianSammit**

### CODE REVIEW

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello Team,

:pencil: Friendly reminder: Remember it's important to have a professional repo so you can increase your job opportunities.
Please take a step back and read the lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
Sadly, for now, your review has been marked as **invalid** because of the following reasons:
- [ ] You are missing a short summary of your changes in the pull request message, please edit it.
- [ ] For next time, remember to write your commits in the imperative tense, as stated in [the article pointed in pathwright](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)

Once you correct this, please submit a new code review and we will be glad to check it :)

### ⚠️  Status: Your project has been marked as invalid ⚠️ 
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Comment by BrianSammit** [comment]
We make a project consist in create a grayscale heat map of the website Smashing Magazine that indicates which elements have the greatest weight in the visual hierarchy.
We use HTML and CSS to go through the project.
For positioning the items we use display flex and we use a different class of color grey.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+          <li><a href="#">Apps</a></li>
+          <li><a href="#">Apps</a></li>
+          <li><a href="#">Apps</a></li>
+          <li><a href="#">Apps</a></li>
+          <li><a href="#">Apps</a></li>
+          <li><a href="#">Apps</a></li>
+          <li><a href="#">Apps</a></li>
+          <li><a href="#">Apps</a></li>
+          <li><a href="#">Apps</a></li>
+          <li><a href="#">Apps</a></li>
```
Comment (expjazz) [inline index.html]: - [ ] This should be centralized.

image


| Yours | Original |
|:---:|:---:|
|image | image |

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: ]

### Well done team :clap:

This is a very nice project that you have done. Your CSS is very organized, and I can see that you went the extra mile using the original content :1st_place_medal: 

However, there is one requirement that your project did not meet:

- [ ] The footer should be centered 

As I said, you did great work here :100: 

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by kitpao — APPROVED** [review]
# 👋 Hello Team,

🌟 Great job fixing the footer 🥳 You have complied with the changes required by previous TSE.

**_Note:_** Please update your live demo link in your readme, it is displaying the page without your last fix

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Pull request merged**